daemon/algod/api/algod.oas2.json

```diff
@@ -3681,8 +3681,7 @@
             "description": "The minimum transaction fee (not per byte) required for the\ntxn to validate for the current network protocol.",
             "type": "integer"
           }
-        },
-        "x-go-package": "github.com/algorand/go-algorand/daemon/algod/api/spec/v1"
```

Comment:
I'm not sure what this was being used for in the v2 `/params` path spec, but it looked unnecessary so I deleted it.

Reply:
I also looked and I'm unsure how `x-go-package` is used.  In addition to general googling (e.g. https://swagger.io/specification/#specification-extensions), I reviewed these sources without finding a reference: 
* https://github.com/algorand/oapi-codegen#extensions
* https://github.com/deepmap/oapi-codegen/blob/master/README.md#extensions

Reply:
As an additional sanity check, I confirmed Go SDK source code generation does _not_ change.

Reply:
It looks like the generator does not use them either: https://github.com/algorand/generator/blob/master/src/main/java/com/algorand/sdkutils/generators/OpenApiParser.java#L129